### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by TheLastMarfist**
This explains how to factor in your head with simple quadratic equations.

Replace this sentence with a brief description of the awesome changes your PR has. 😊

---

## ✅️ By submitting this PR, I have verified the following:

- [ ] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md`, since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Additional PR title examples:
    - **Git: edit Git Commit article**
    - **PHP: create PHP section and add Data Structures article**
- [ ] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md).
- [ ] No plagiarized, duplicate, or repetitive content that has been directly copied from another source.

### CODE REVIEW

File: src/pages/mathematics/factoring-in-algebra/index.md [hunk 1]
```diff
@@ -3,13 +3,19 @@ title: Factoring in Algebra
 ---
 ## Factoring in Algebra
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/mathematics/factoring-in-algebra/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Factoring is how you take a quadratic in standard form and put it in factored form. The standard form for a quadratic is normally shown as ax<sup>2</sup> + bx + c where a and b are coefficients and c is a constant. Factored form is normally shown as (x - d)(x - e) where d and t are both coefficients.
```
Comment (ezioda004) [inline src/pages/mathematics/factoring-in-algebra/index.md]: Hi, shouldn't it be (x - d)(x - t) as you've said d and t are coefficients.

File: src/pages/mathematics/factoring-in-algebra/index.md [hunk 2]
```diff
@@ -3,13 +3,19 @@ title: Factoring in Algebra
...
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+## Factoring in Your Head
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+This method of factoring usually works best when a = 1. To do this when a = 1 you look at what two numbers add to equal b while also multiplying to c. Those two numbers will then be equal to d and t.
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+Example 1:<br>
```
Comment (ezioda004) [inline src/pages/mathematics/factoring-in-algebra/index.md]: You can make Example1 as heading like: 
#### Example1: using `#### Example1:`
It makes it more readable and formatted.

File: src/pages/mathematics/factoring-in-algebra/index.md [hunk 3]
```diff
@@ -3,13 +3,19 @@ title: Factoring in Algebra
...
+x<sup>2</sup> + 2x + 1<br>
+Here you have to look for two numbers that multiply to 1 and add to 2. Those two numbers are 1 and 1. Then you put those numbers into the factored form like this:<br>
+(x + 1)(x + 1)
 
+Example 2:<br>
```
Comment (ezioda004) [inline src/pages/mathematics/factoring-in-algebra/index.md]: Likewise for Example 2.

**Comment by QuincyLarson** [comment]
@ezioda004 Thanks for the assist! I'm going to go ahead and merge this. 

@TheLastMarfist Thanks for adding this explanation!

**Pull request merged**